backend/Controller.js

```diff
@@ -87,9 +87,55 @@ exports.loginUser = async (req, res) => {
 
   //set new access token
   spotifyApi.refreshAccessToken().then(
-    (data) => {
+    async (data) => {
       spotifyApi.setAccessToken(data.body.access_token);
 
+      let topTracks = [];
+
+      //get user's top 3 tracks
+      await spotifyApi
+        .getMyTopTracks()
+        .then((data) => {
+          for (let i = 0; i < 3; i++) {
+            let track = {};
+            track.songName = data.body.items[i].name;
+            track.songID = data.body.items[i].id;
+            track.songPopularity = data.body.items[i].popularity;
+            track.linkURL = data.body.items[i].external_urls.spotify;
+            track.imageURL = data.body.items[i].album.images[0].url;
+            track.artistName = data.body.items[i].artists[0].name;
+
```

Comment:
Maybe take the artist id as well. You should be able to use:
"track.artistID = data.body.items[i].artists[0].id;" 
and then also add it in the model I guess

Reply:
idk if what I did has any difference to just doing a comment but oh well

Reply:
When would we need to artist ID for the song?